packages/Input/src/Input.tsx

```diff
@@ -143,11 +168,25 @@ const Input: React.FunctionComponent<InputProps> = React.forwardRef(
       </span>
     );
 
-    return prefixIcon || suffixIcon ? (
+    return prefixIcon || suffixIcon || displayCharIndicator ? (
       <div className="ids-input__wrapper">
         {prefixIcon && addPrefixIcon}
         {inputRender}
-        {suffixIcon && addSuffixIcon}
+        {suffixIcon || displayCharIndicator ? (
+          <div
+            className={cx('ids-input__asset-wrapper', {
+              'ids-input__asset-wrapper--has-char-indicator':
+                displayCharIndicator,
+            })}
+          >
+            {suffixIcon && addSuffixIcon}
+            {displayCharIndicator && (
+              <div className="ids-input__char-indicator">
+                {inputMaxLength - charLength}
```

Comment:
Their is something with the character count that act strange. For some reason the length of `😀😀` is the same as `😀😀a`

Visually:
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/4323800/f82f9a7f-0950-4afc-bffb-32fab2ad32be)
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/4323800/5d8b56b8-fd87-4bbc-8900-f95c9198707d)

Reply:
It seems like the original fix we had for emojis is no longer needed. It was now causing issues.